### ISSUE TITLE
8.2 to 9.0 : Lost calendar share

### ISSUE THREAD

**Issue by philippe-opendsi** [owncloud/core#23611]
Hi,
I've done the upgrade from owncloud 8.2.3 to 9.0. After the upgrade, I can't see the previously existing shared calendar from other users

No error message during the migration done with occ update

```
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Set log level to debug
Turned on maintenance mode
Checking whether the database schema can be updated (this can take a long time depending on the database size)
Checked database schema update
Checking updates of apps
Checking whether the database schema for <activity> can be updated (this can take a long time depending on the database size)
Checking whether the database schema for <files_sharing> can be updated (this can take a long time depending on the database size)
Checking whether the database schema for <files_trashbin> can be updated (this can take a long time depending on the database size)
Checked database schema update for apps
Updating database schema
Updated database
Disabled 3rd-party app: calendar
Disabled 3rd-party app: contacts
Disabled 3rd-party app: files_videoviewer
Updating <files_pdfviewer> ...
Updated <files_pdfviewer> to 0.8
Updating <files_texteditor> ...
Updated <files_texteditor> to 2.1
Updating <files> ...
Updated <files> to 1.4.4
Updating <activity> ...
Updated <activity> to 2.2.1
Updating <files_sharing> ...
Updated <files_sharing> to 0.9.1
Updating <files_trashbin> ...
Updated <files_trashbin> to 0.8.0
Updating <files_versions> ...
Updated <files_versions> to 1.2.0
Updating <provisioning_api> ...
Updated <provisioning_api> to 0.4.1
Update 3rd-party app: calendar
Update 3rd-party app: contacts
Starting code integrity check...
Finished code integrity check
Update successful
Turned off maintenance mode
Reset log level
```
Server configuration
Operating system: Ubuntu 14.04
Web server:nginx
Database:Mariadb
PHP version: 5.5.9

Does anyone have the same issue ?

Thanks for your help

Philippe

**Comment by DeepDiver1975** [comment]
Shared with group or a single user?

**Comment by philippe-opendsi** [comment]
I think both. I have to ask to the users. Is there a way to see that in the database ?

**Comment by DeepDiver1975** [comment]
All shared calendar are listed in OC_DAV_shares

**Comment by DeepDiver1975** [comment]
We fixed some issues with 9.0.1 with respect to calendar sharing

**Comment by philippe-opendsi** [comment]
I confirm, calendar are shared with group and single user. I will make test with 9.01. Thanks

**Comment by DeepDiver1975** [comment]
> I will make test with 9.01

You might want to help testing
https://github.com/owncloud/core/pull/23552
https://github.com/owncloud/core/pull/23318

THX

**Comment by PVince81** [comment]
The fixes made it into 9.0.1, please try again with that version.
